include/seastar/core/deleter.hh

```diff
@@ -193,6 +193,8 @@ SEASTAR_MODULE_EXPORT_END
 struct free_deleter_impl final : deleter::impl {
     void* obj;
     free_deleter_impl(void* obj) : impl(deleter()), obj(obj) {}
+    free_deleter_impl(const free_deleter_impl&) = delete;
+    free_deleter_impl(free_deleter_impl&&) = delete;
     virtual ~free_deleter_impl() override { std::free(obj); }
```

Comment:
As soon as you have a virtual destructor, the genereated copy/move stuff is disabled.

Reply:
https://i.stack.imgur.com/b2VBV.png

Declared destructor disables only move-stuff

Reply:
Oops! I see there was a proposal to deprecate copy construction/assignment, if a destructor is defined, but nothing done.